spec/lex.dd

```diff
@@ -265,8 +290,9 @@ $(GNAME StringLiteral):
 
     $(GLINK HexString)
     $(GLINK DelimitedString)
-    $(GLINK TokenString)
+    $(GLINK TokenString))
```

Comment:
The closing brace should appear on the next line s.t. it's consistent with the other rules.